### PR TITLE
(patch): Add line-height styling to Spinner

### LINKED ISSUE #24634: [Bug]: Spinner has wrong size because of line-height
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
docs site
```

### Are you reporting Accessibility issue?

no

### Reproduction

https://react.fluentui.dev/?path=/docs/components-spinner--default

## Actual Behavior
Spinner height is higher due to default line-height being applied

## Expected Behavior
Spinner has equal width and height

image

with line-height explicitly set to 0:
image

### Logs

_No response_

### Requested priority

Blocking

### Products/sites affected

Teams

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### PR DESCRIPTION
**Pull request by tomi-msft**
Any parent component to Spinner that has `line-height` styling will cause the `Spinner` to have the wrong size. This PR adds styling to `Spinner` that makes `lineHeight: '0'`

Fixes #24634

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/0ff310ef1922f08eaa8c9b801c1dbfe874e1b15f'>0ff310ef1922f08eaa8c9b801c1dbfe874e1b15f</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24643/perf-test-react-components/Avatar-mount_base.html">1268</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24643/perf-test-react-components/Avatar-mount.html">1254</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24643/perf-test-react-components/Button-mount_base.html">930</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24643/perf-test-react-components/Button-mount.html">940</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24643/perf-test-react-components/FluentProvider-mount_base.html">1486</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24643/perf-test-react-components/FluentProvider-mount.html">1471</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24643/perf-test-react-components/FluentProviderWithTheme-mount_base.html">570</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24643/perf-test-react-components/FluentProviderWithTheme-mount.html">592</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24643/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">533</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24643/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">557</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24643/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">564</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24643/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">597</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24643/perf-test-react-components/MakeStyles-mount_base.html">1959</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24643/perf-test-react-components/MakeStyles-mount.html">1955</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24643/perf-test-react-components/SpinButton-mount_base.html">2318</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24643/perf-test-react-components/SpinButton-mount.html">2347</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: packages/react-components/react-spinner/src/components/Spinner/useSpinnerStyles.ts [hunk 1]
```diff
@@ -255,7 +256,7 @@ const useTrackStyles = makeStyles({
 const useLabelStyles = makeStyles({
   // style for label
   inverted: {
-    color: tokens.colorNeutralForegroundStaticInverted,
+    color: 'rgba(255, 255, 255, 0.2)', // This is white alpha but the token is not exported
```
Comment (spmonahan) [inline packages/react-components/react-spinner/src/components/Spinner/useSpinnerStyles.ts:259]: Should the label color have an alpha applied? Won't this make the label nearly invisible?

Comment (tomi-msft) [reply packages/react-components/react-spinner/src/components/Spinner/useSpinnerStyles.ts:259]: This is for the inverted styling. The `colorNeutralForegroundStaticInverted` token doesn't exist, so the guidance was to use white alpha

**Review by khmakoto — APPROVED** [review]

**Pull request merged**